pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/ClusterChangeMediator.java

```diff
@@ -116,10 +117,20 @@ public void run() {
     };
   }
 
-  private void processClusterChange(ChangeType changeType, ClusterChangeHandler changeHandler) {
+  private void processClusterChange(ChangeType changeType, List<ClusterChangeHandler> changeHandlers) {
     long startTime = System.currentTimeMillis();
     LOGGER.info("Start processing {} change", changeType);
-    changeHandler.processClusterChange();
+    for (ClusterChangeHandler changeHandler : changeHandlers) {
+      try {
+        long handlerStartTime = System.currentTimeMillis();
+        changeHandler.processClusterChange(changeType);
+        LOGGER.info("Finish handling {} change for handler: {} in {}ms", changeType, changeHandler.getClass().getName(),
```

Comment:
I would introduce a method in ClusterChangeHandler to get the name.

Reply:
I think the class name itself should be enough, no need to complicate the interface